loki/loki-quickstart/step1.md

```diff
@@ -1,49 +1,56 @@
 
-    With `evaluate-loki` as the current working directory, start the demo environment using `docker compose`:
+   ```bash
+   docker compose up -d
+   ```{{execute}}
 
-    ```bash
-    docker-compose up -d
-    ```{{execute}}
+   You should see something similar to the following:
 
-    You should see something similar to the following:
+   ```bash
+   ✔ Network evaluate-loki_loki          Created      0.1s
+   ✔ Container evaluate-loki-minio-1     Started      0.6s
+   ✔ Container evaluate-loki-flog-1      Started      0.6s
+   ✔ Container evaluate-loki-backend-1   Started      0.8s
+   ✔ Container evaluate-loki-write-1     Started      0.8s
+   ✔ Container evaluate-loki-read-1      Started      0.8s
+   ✔ Container evaluate-loki-gateway-1   Started      1.1s
+   ✔ Container evaluate-loki-grafana-1   Started      1.4s
+   ✔ Container evaluate-loki-alloy-1     Started      1.4s
+   ```
 
-    ```bash
-    ✔ Network evaluate-loki_loki          Created      0.1s 
-    ✔ Container evaluate-loki-minio-1     Started      0.6s 
-    ✔ Container evaluate-loki-flog-1      Started      0.6s 
-    ✔ Container evaluate-loki-backend-1   Started      0.8s 
-    ✔ Container evaluate-loki-write-1     Started      0.8s 
-    ✔ Container evaluate-loki-read-1      Started      0.8s 
-    ✔ Container evaluate-loki-gateway-1   Started      1.1s 
-    ✔ Container evaluate-loki-grafana-1   Started      1.4s 
-    ✔ Container evaluate-loki-alloy-1     Started      1.4s
-    ```
+1. (Optional) Verify that the Loki cluster is up and running.
 
-4. (Optional) Verify that the Loki cluster is up and running.
-    - The read component returns `ready` when you point a web browser at [http://localhost:3101/ready]({{TRAFFIC_HOST1_3101}}/ready). The message `Query Frontend not ready: not ready: number of schedulers this worker is connected to is 0` will show prior to the read component being ready.
-    - The write component returns `ready` when you point a web browser at [http://localhost:3102/ready]({{TRAFFIC_HOST1_3102}}/ready). The message `Ingester not ready: waiting for 15s after being ready` will show prior to the write component being ready.
+   - The read component returns `ready` when you browse to [http://localhost:3101/ready](http://localhost:3101/ready).
+     The message `Query Frontend not ready: not ready: number of schedulers this worker is connected to is 0` shows until the read component is ready.
+   - The write component returns `ready` when you browse to [http://localhost:3102/ready](http://localhost:3102/ready).
+     The message `Ingester not ready: waiting for 15s after being ready` shows until the write component is ready.
 
-5. (Optional) Verify that Grafana Alloy is running.
-    - Grafana Alloy's UI can be accessed at [http://localhost:12345]({{TRAFFIC_HOST1_12345}}).  
+1. (Optional) Verify that Grafana Alloy is running.
+   - You can access the Grafana Alloy UI at [http://localhost:12345](http://localhost:12345).
```

Comment:
I think this is already in your feature notes but thought I would flag it in the review so its there